### PR TITLE
Add versioning logging

### PR DESCRIPTION
**Pull request by gosusnp**
## What
* Log which protocol version we use

### CODE REVIEW

**Comment by colesnodgrass** [comment]
Should we also put this information into the tracing spans?

File: airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/VersionedAirbyteMessageBufferedWriterFactory.java [hunk 1]
```diff
@@ -25,6 +29,11 @@ public VersionedAirbyteMessageBufferedWriterFactory(final AirbyteMessageSerDePro
 
   @Override
   public AirbyteMessageBufferedWriter createWriter(BufferedWriter bufferedWriter) {
+    final boolean needMigration = !protocolVersion.getMajorVersion().equals(migratorFactory.getMostRecentVersion().getMajorVersion());
+    LOGGER.info(
```
Comment (benmoriceau) [inline airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/VersionedAirbyteMessageBufferedWriterFactory.java:33]: This class is a factory so I am assuming that it is called only once per sync and not once per message. Could you confirm that? If not, we need to switch this log to debug to ensure that we won't generate gigantic logs.

**Review by benmoriceau — APPROVED** [review]
LGTM, I'll let you double check the comment I have put but I am assuming it is ok.

File: airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/VersionedAirbyteMessageBufferedWriterFactory.java (hunk 1 above)
Comment (gosusnp) [reply airbyte-commons-worker/src/main/java/io/airbyte/workers/internal/VersionedAirbyteMessageBufferedWriterFactory.java:33]: Yes, this is called once per sync.

**Comment by gosusnp** [comment]
@colesnodgrass, I don't think it would hurt us to have it in the spans, however, I would add it from a different place. Probably where we create the IntegrationLauncher, will do in a different PR.

**Pull request merged**